.github/workflows/react-components.yml

```diff
@@ -29,8 +30,10 @@ jobs:
         working-directory: .
       - name: build
         run: npm run build:storybook && npm run build
+      - name: Setup kernel for Karma tests, increase watchers
+        run: echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
       - name: test
-        run: npm run test:coverage
```

Comment:
I think this is a mistake from the merge.

Reply:
There is nothing here, maybe you looked at this in the outdated version?

Reply:
Maybe it's because I did a merge main, `run: npm run test:coverage` has been changed back to `run: npm run test`, we should be using the coverage script.

Reply:
Oh sorry, I misunderstood what you told me =/. Yeah, the `npm run test` now has the coverage too. I can create another karma file run all test (without the coverage) and switch the name to the one that we have now to `test:coverge`